haskell/private/create_dummy_archive.bzl

```diff
@@ -0,0 +1,63 @@
+load("@bazel_skylib//:lib.bzl", "paths")
+load(":cc.bzl", "cc_interop_info")
+load(":private/context.bzl", "haskell_context")
+
+def _impl(ctx):
+  """Create empty archive so that GHC has some input files to work on during
+  linking.
+
+  See: https://github.com/facebook/buck/blob/126d576d5c07ce382e447533b57794ae1a358cc2/src/com/facebook/buck/haskell/HaskellDescriptionUtils.java#L295
+
+  Returns:
+    File, the created dummy archive.
+  """
+
+  # TODO: this is a duplication from ./haskell_impl.bzl
+  hs = haskell_context(ctx)
+  cc = cc_interop_info(ctx)
```

Comment:
Also this, but I think that can’t be circumvented in this case.

Reply:
I don't see where this `cc` thing is used at all, drop it?

Reply:
We're generating a dummy module and then compile it, why should we be concerned with cc interop in this case?